tools/myst/config.h

```diff
@@ -6,6 +6,7 @@
 
 #include <myst/json.h>
 #include <myst/kernel.h>
+#include <openenclave/bits/defs.h>
```

Comment:
Is this needed?

Reply:
yes, for OE_PAGE_SIZE used in the start_address value check.

Reply:
I think mystikos has its own PAGE_SIZE definition. Can you check?

Reply:
I see PAGE_SIZE = OE_PAGE_SIZE = 0x1000. Changed code to use PAGE_SIZE instead.